shipyard-controller/handler/shipyardcontroller_utils.go

```diff
@@ -125,31 +82,6 @@ func GetTaskSequencesByTrigger(eventScope models.EventScope, completedTaskSequen
 	return result
 }
 
-func GetMergedPayloadForSequenceTriggeredEvent(inputEvent *models.Event, eventPayload map[string]interface{}, eventHistory []interface{}) (interface{}, error) {
```

Comment:
this function is now offered as a method of the SequenceExecution model. This should help to separate the concerns